vcd/resource_vcd_vapp_vm.go

```diff
@@ -69,7 +66,14 @@ func resourceVcdVAppVm() *schema.Resource {
 				Optional: true,
 				ForceNew: true,
 			},
-
+			"metadata": {
+				Type:     schema.TypeMap,
+				Optional: true,
+				/*
```

Comment:
Nit: please use `//` comment in this case, which would be more compact (two lines as opposed to four without a need to indent the text).

Reply:
Fixed.